package.json

```diff
@@ -7,76 +7,64 @@
     "build": "webpack --mode production",
     "start": "webpack-dev-server --mode development",
     "watch": "webpack -debug --config webpack.config.js --watch --progress --mode development",
-    "lint:js": "eslint src/js/**/*.js",
+    "lint:js": "eslint patientsearch/src/js/*.js patientsearch/src/js/**/*.js; exit 0",
```

Comment:
Would it be helpful if I set up `eslint` to run on CI (in case you forget to run it locally) in a separate PR?

There are [some github actions](https://github.com/ataylorme/eslint-annotate-action) that can add "annotations" to PRs with the output of eslint

Reply:
@ivan-c It would be really helpful, but not sure how hard this will be?